aiida_pseudo/cli/install.py

```diff
@@ -108,9 +109,25 @@ def download_sssp(
     from aiida_pseudo.groups.family import SsspFamily
     from .utils import attempt
 
-    url_sssp_base = 'https://legacy-archive.materialscloud.org/file/2018.0001/v4/'
-    url_archive = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'tar.gz')}"
-    url_metadata = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'json')}"
+    url_template = 'https://archive.materialscloud.org/record/file?filename={filename}&parent_id=19'
```

Comment:
Thanks for the changes. Everything looks good to me now, except I still have a question about this magic `parent_id=19`. I have been trying to figure out how it relates to the main archive entry. I take it it refers to the actual archive record: https://archive.materialscloud.org/record/2021.76
As you can see, there in the URL it uses an ID that seems based on the year and some auto-incrementing number. But this ID changes if a new version of the SSSP record is created. The question is whether the `parent_id=19` is persistent and doesn't change. I.e., when a `v8` will be released with the next patch version of `1.1`, i.e. `1.1.3`, will the URL still work?

Reply:
The parent_id is persistent within versions of the same record, meaning that it does not change if a new version of a record is created. 
If you create a new SSSP record that is not a new version of https://archive.materialscloud.org/record/2021.76 then the parent_id will be different.

Reply:
OK Thanks @vgranata that should be fine. Then we can almost merge this @mbercx . Only thing is that we should probably add the patch version to the description of the group. Otherwise it will only be possible to know what patch version a certain family is by looking at the md5 of the tarball that was downloaded, which is contained in the description. I think it would make sense to store the patch at the very least in the description of the family. And then the tests should be fixed of course, because they are failing